app/controllers/users_controller.rb

```diff
@@ -19,9 +28,62 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by params[:id]
+    if @user.edit user_params
+      flash.now[:success] = "User found"
+      render :edit
+    else
+      flash[:danger] = "User not found"
+      redirect_to root_url
+    end  
+  end
+
+  def update
+    @user = User.find_by params[:id]
```

Comment:
because you loaded user by using load_user before_action
so dont need this line, remove it please

Reply:
done